include/zephyr/sys/util_macro.h

```diff
@@ -48,6 +48,17 @@ extern "C" {
 /** @brief 64-bit unsigned integer with bit position @p _n set. */
 #define BIT64(_n) (1ULL << (_n))
 
+/**
+ * @brief Toggle a bit
```

Comment:
```suggestion
 * @brief Toggles a specific bit of a given variable.
```
Would this be more accurate?